gateway/src/apicast/policy/token_introspection/apicast-policy.json

```diff
@@ -20,6 +20,18 @@
       "client_secret": {
         "description": "Client Secret for the Token Introspection Endpoint",
         "type": "string"
+      },
+      "max_ttl_tokens": {
+        "description": "Max TTL for cached tokens",
+        "type": "integer",
+        "minimum": 1,
+        "maximum": 3600
+      },
+      "max_cached_tokens": {
+        "description": "Max number of tokens to cache",
+        "type": "integer",
+        "minimum": 0,
+        "maximum": 10000
```

Comment:
Probably should have "default" 0 ?

Reply:
Right. It already defaults to 0 in the code but should be here too.

Reply:
@davidor probably not needed. Only if we want to populate it in the UI.